distributions/otelcol-contrib/manifest.yaml

```diff
@@ -4,199 +4,199 @@ dist:
   description: OpenTelemetry Collector Contrib
   version: 0.95.1
```

Comment:
This was getting removed in a subsequent PR correct? I thought I had seen discussion related to that already.

Reply:
Thanks for catching this! This is not being removed and it was wrong, I just didn't notice because I replaced `0.95.0` and didn't realize we had made this change